### DISCUSSION TITLE
When you add to Toonsquid for iPhone

### DISCUSSION THREAD

**Discussion by CrtyxScuizx** [keiwando/toonsquid#128] [Ideas]
I wish for iPhone available on Toonsquid

**Comment by Lioncub901** [comment]
I may not be a  genius but I don’t know how one can cram all those features in a tiny device. If he did I think the app would not be so user friendly since there will be many tabs and buttons.

**Reply by kavlor** [reply]
@Lioncub901 
I agree plus if you want this sort of power you'd have a tablet , Flipaclip is perfect for Mobile anyway.Personally I use a budget Android phone then I can afford higher end IOS tablets.
 
Even if you were to say 'well I could use it on the phone when a tablet isn't convenient' people will complain it doesn't work very well on it and not see it as mainly for the Ipad.

Can't speak for the developer but those are my thoughts.
